src/main/java/uk/gov/hmcts/cmc/claimstore/services/TimeService.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.cmc.claimstore.services;
+
+import org.springframework.stereotype.Component;
+
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+
+@Component
+public class TimeService {
```

Comment:
This class is a copy/paste of `uk.gov.hmcts.cmc.domain.utils.LocalDateTimeFactory` in `domain-model` module. Please remove this duplication.

Additionally, please provide unit tests for this service. And since it's a service, please use `@Service` instead of `@Component` (it's effectively the same, but make more sense when you read it).